i18n/fr/docusaurus-plugin-content-docs/current/wallet/Wallet-Guide.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar_position: 10
+title: Full Node Wallet Guide
+---
+
+> ## ⚠️ Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page.
+
+The wallet API can be called using our Swagger UI at [http://127.0.0.1:12973/docs](http://127.0.0.1:12973/docs) or by using `curl`. Make sure that your full node is running so you could access the Swagger UI.
+
+## Create a new wallet
+
+You can create a new wallet by doing a POST with the following data on `/wallets`.
+
+```json
```

Comment:
If we change this to `js`, then the the code highlighter won't complain with a red background (json cannot have comments, js can.
```suggestion
```js
```

Reply:
i trust you on this :)

Reply:
There's A LOT of places where we use ```json, though...

Reply:
If you wanna use comments inside the json block, use js. Otherwise json.

Reply:
ok, but you are suggesting to add js on a json block?

Reply:
aaah sorry! i get you now... need more coffee